nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -948,7 +935,36 @@ def permuted_ols(
     return outputs
 
 
-def _check_inputs_permuted_ols(n_jobs, tfce, masker, threshold, target_vars):
+def _make_array_contiguous(array):
+    """Make arrays contiguous for code efficiency."""
+    if not array.flags["C_CONTIGUOUS"]:
+        # useful to developer
+        warnings.warn("Target variates not C_CONTIGUOUS.")
+        array = np.ascontiguousarray(array)
+    return array
+
+
+def _compute_t_stat_threshold(
```

Comment:
one-line docstrings are always welcome for this type of functions

Reply:
done 
and replaced dof by degrees_of_freedom